### PR TITLE
Add some tests for named properties on Storage.

### PR DESCRIPTION
**Pull request by Ms2ger**

### CODE REVIEW

File: webstorage/set.window.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+            assert_equals(storage[key], undefined);
+            assert_equals(storage.getItem(key), null);
+            assert_equals(storage[key] = value, value);
+            assert_equals(storage[key], "value");
+            assert_equals(storage.getItem(key), "value");
+        }, "Setting property with toString for key " + key + " on " + name);
+
+        test(function() {
+            Storage[key] = "proto";
```
Comment (jdm) [inline webstorage/set.window.js]: The capital S looks wrong; isn't this undefined? What am I missing?

**Review by jdm — CHANGES_REQUESTED** [review]

File: webstorage/set.window.js (hunk 1 above)
Comment (Ms2ger) [reply webstorage/set.window.js]: A `.prototype` I didn't write. Added, and then fixed all the bugs I missed.

**Review by jdm — APPROVED** [review]

File: webstorage/defineProperty.window.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+["localStorage", "sessionStorage"].forEach(function(name) {
+    [9, "x"].forEach(function(key) {
```
Comment (inexorabletash) [inline webstorage/defineProperty.window.js]: Test with a Symbol as well? That's out of scoped for "named properties" though. Separate test?

File: webstorage/defineProperty.window.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+["localStorage", "sessionStorage"].forEach(function(name) {
```
Comment (inexorabletash) [inline webstorage/defineProperty.window.js]: Style: how about using arrow functions to reduce the "function" spam?

File: webstorage/defineProperty.window.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+            var storage = window[name];
+            storage.clear();
+
+            assert_equals(storage[key], undefined);
+            assert_equals(storage.getItem(key), null);
+            assert_equals(Object.defineProperty(storage, key, desc), storage);
+            assert_equals(storage[key], "value");
+            assert_equals(storage.getItem(key), "value");
+        }, "Defining data property for key " + key + " on " + name);
```
Comment (inexorabletash) [inline webstorage/defineProperty.window.js]: Style: could use \`template strings with ${interpolations}\`

**Review by inexorabletash — APPROVED** [review]
LGTM with some style suggestions, but nothing required.

**Comment by Ms2ger** [comment]
Made a note to write Symbol tests at some point, and left the modern-js nits alone for now?

**Pull request merged**